driverconfig/config.yaml

```diff
@@ -116,6 +116,8 @@ drivers:
       - configVersion: v2.6.0
         useDefaults: true
         supportedVersions:
+          - version: v121
+          - version: v122
```

Comment:
@nitesh3108 , have you tested driver install via operator in Tanzu. If not recommend not to add support for K8s 1.22

Reply:
yes tested

Reply:
As per the documentation in https://dell.github.io/csm-docs/docs/csidriver/partners/tanzu/
" Installation of CSI drivers in Tanzu via Operator has not been qualified."

Reply:
Let me revert then